src/mgmt/srv_drpc.c

```diff
@@ -1579,9 +1562,9 @@ ds_mgmt_drpc_pool_query(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 
 	len = mgmt__pool_query_resp__get_packed_size(&resp);
 	D_ALLOC(body, len);
-	if (body == NULL) {
+	if (body == NULL)
 		drpc_resp->status = DRPC__STATUS__FAILED_MARSHAL;
-	} else {
```

Comment:
Same note as above on the curly braces.

Reply:
will address in https://github.com/daos-stack/daos/pull/3423 if that's okay

Reply:
https://github.com/daos-stack/daos/pull/3423/commits/74c935f2e90f268555cd656beb4e4593d2d0955b